src/components/cards/verticalCard/VerticalCard.jsx

```diff
@@ -23,22 +28,43 @@ function VerticalCard({ video }) {
             className="card-img videolib-verticalcard-img-resize"
             src={img}
           />
-          {watchLaterVideos.find((item) => item._id === _id) ? (
+          
+          {watchLaterVideos?.find((item) => item._id === _id) ? (
             <i
-              className="fa-solid fa-clock position-abs verticalCard-watchLater-icon-position verticalCard-watchLater-icon-select-clr"
+              className="fa-solid fa-clock position-abs verticalCard-watchLater-icon verticalCard-watchLater-icon-select-clr"
               onClick={() => handleRemoveFromWatchLater(_id, token)}
             ></i>
           ) : (
             <i
-              className="fa-solid fa-clock position-abs verticalCard-watchLater-icon-position"
+              className="fa-solid fa-clock position-abs verticalCard-watchLater-icon"
               onClick={() => handleAddToWatchLater(video, token)}
             ></i>
           )}
 
+          <i
+            onClick={() => setShowPlaylistDropdown((prev) => !prev)}
+            className="fa-solid fa-file-circle-plus position-abs verticalCard-playlist-icon"
+          ></i>
+
+          {showPlaylistDropdown ? (
+            <PlaylistDropdown
+              setShowPlaylistDropdown={setShowPlaylistDropdown}
+              video={video}
+              hideSaveToPlaylist={false}
+            />
+          ) : (
+            ""
```

Comment:
you can use `&&` for showing element if it has no else condition.

Reply:
Yess I will take these suggestion in next pr